sceptre/scipool/config/bmgfki/sc-product-s3-private-enc.yaml

```diff
@@ -55,7 +55,11 @@ sceptre_user_data:
       Info:
         LoadTemplateFromURL:  'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/{{stack_group_config.service_catalog_library}}/v1.1.33/s3/sc-s3-encrypted-ra.yaml'
       Name: 'v1.1.33'
-    - Description:  'Disable bucket ACLs. {{ range(1, 10000) | random }}'
+    - Description:  'Disable bucket ACLs.'
       Info:
         LoadTemplateFromURL:  'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/{{stack_group_config.service_catalog_library}}/v1.1.38/s3/sc-s3-encrypted-ra.yaml'
       Name: 'v1.1.38'
+    - Description:  'Update cfn-cr-same-region-bucket-download python version. {{ range(1, 10000) | random }}'
+      Info:
+        LoadTemplateFromURL:  'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/{{stack_group_config.service_catalog_library}}/v1.3.8/s3/sc-s3-encrypted-ra.yaml'
+      Name: 'v1.3.8'
```

Comment:
this looks wrong because the last version was `v1.1.38`

Reply:
That's how long it's been since we've updated this particular template. The tagged commit with the update we want is https://github.com/Sage-Bionetworks/service-catalog-library/tree/v1.3.8

Reply:
sorry, the numbers are so similar that i got mixed up. LGTM now.